projects/igniteui-angular/src/lib/grids/grid/column-resizing.spec.ts

```diff
@@ -719,6 +720,29 @@ describe('IgxGrid - Deferred Column Resizing', () => {
         fixture.detectChanges();
         expect(column.width).toEqual('111px');
     }));
+
+    it('should size headers correctly when column width is below the allowed minimum.', fakeAsync(() => {
+        const fixture = TestBed.createComponent(ColGridComponent);
+        fixture.detectChanges();
+
+        const headers = fixture.debugElement.queryAll(By.css(COLUMN_HEADER_CLASS));
+        const headerGroups = fixture.debugElement.queryAll(By.css(COLUMN_HEADER_GROUP_CLASS));
+        const filteringCells = fixture.debugElement.queryAll(By.css('igx-grid-filtering-cell'));
```

Comment:
Could you please extract `igx-grid-filtering-cell` class as constant similar to the ones above.